functional/sev/run.sh

```diff
@@ -51,6 +51,7 @@ install_sevctl_and_export_sev_cert_chain() {
 
 run_kbs() {
   git clone https://github.com/confidential-containers/simple-kbs.git --branch main
+  (cd simple-kbs && git checkout -b branch_0.1.1 0.1.1)
```

Comment:
Could we add the url to simple-kbs and its version in the `versions.yaml`?

Reply:
I've added simple-kbs to the `versions.yaml`. However, the method `get_version` fails as it requires sourcing a script from the `kata-containers` repo that it expects to be in the GOPATH. `kata-containers` repo isn't there.

I think the `lib.sh` files should be cleaned up so that they aren't dependent like that across repos. Also, we should separate common bash functionality with items that say need golang, code cloning (installing yq) or apt package installations. It would make utilizing these scripts much easier for different cases.

For now, I've copied a modified version of these methods directly into my script to get it to work. We can move them later to the appropriate place when we've decided how to organize the bash libs better.

@wainersm does this sound okay?